src/node/entities.h

```diff
@@ -60,50 +60,68 @@ namespace ccf
-    static constexpr auto VALUES = "public:ccf.gov.values";
-    static constexpr auto CONSENSUS = "public:ccf.gov.consensus";
-    static constexpr auto WHITELISTS = "public:ccf.gov.whitelists";
+    static constexpr auto NODE_CODE_IDS = "public:ccf.gov.nodes.code_ids";
+
+    // Service information
+    static constexpr auto SERVICE = "public:ccf.gov.service";
+    static constexpr auto CONFIGURATION = "public:ccf.gov.service.config";
+    static constexpr auto VALUES = "public:ccf.internal.values";
+
+    // Governance
     static constexpr auto PROPOSALS = "public:ccf.gov.proposals";
     static constexpr auto GOV_SCRIPTS = "public:ccf.gov.governance.scripts";
-    static constexpr auto APP_SCRIPTS = "public:ccf.gov.app_scripts";
-    static constexpr auto MODULES = "public:ccf.gov.modules";
-    static constexpr auto SECRETS = "public:ccf.gov.secrets";
-    static constexpr auto NODE_CODE_IDS = "public:ccf.gov.nodes.code_ids";
     static constexpr auto GOV_HISTORY = "public:ccf.gov.governance.history";
-    static constexpr auto SERVICE = "public:ccf.gov.service";
-    static constexpr auto SHARES = "public:ccf.gov.shares";
-    static constexpr auto CONFIGURATION = "public:ccf.gov.config";
-    static constexpr auto SUBMITTED_SHARES = "public:ccf.gov.submitted_shares";
-    static constexpr auto SNAPSHOT_EVIDENCE =
-      "public:ccf.gov.snapshot_evidence";
-    static constexpr auto CA_CERT_DERS = "public:ccf.gov.ca_cert_ders";
-    static constexpr auto JWT_ISSUERS = "public:ccf.gov.jwt_issuers";
+    static constexpr auto WHITELISTS = "public:ccf.gov.whitelists";
+
+    // JS applications
+    static constexpr auto APP_SCRIPTS = "public:ccf.app.scripts";
+    static constexpr auto MODULES = "public:ccf.app.modules";
+    static constexpr auto ENDPOINTS = "public:ccf.app.endpoints";
+
+    // JWT issuers
+    static constexpr auto CA_CERT_DERS = "public:ccf.gov.jwt.ca_certs_der";
+    static constexpr auto JWT_ISSUERS = "public:ccf.gov.jwt.issuers";
     static constexpr auto JWT_PUBLIC_SIGNING_KEYS =
-      "public:ccf.gov.jwt_public_signing_keys";
+      "public:ccf.gov.jwt.public_signing_keys";
     static constexpr auto JWT_PUBLIC_SIGNING_KEY_ISSUER =
-      "public:ccf.gov.jwt_public_signing_key_issuer";
-    static constexpr auto ENDPOINTS = "public:ccf.gov.endpoints";
+      "public:ccf.gov.jwt.public_signing_key_issuer";
 
+    // Internal only
+    static constexpr auto SECRETS = "public:ccf.internal.secrets";
+    static constexpr auto SHARES = "public:ccf.internal.shares";
+    static constexpr auto SUBMITTED_SHARES =
+      "public:ccf.intenal.encrypted_submitted_shares";
+    static constexpr auto SNAPSHOT_EVIDENCE =
+      "public:ccf.internal.snapshot_evidence";
     static constexpr auto SIGNATURES = "public:ccf.internal.signatures";
 
+    // Consensus
+    static constexpr auto CONSENSUS = "public:ccf.internal.consensus";
+    static constexpr auto AFT_REQUESTS = "ccf.internal.consensus.requests";
```

Comment:
This was public, which didn't seem right. @ashamis?

Reply:
I did not realize this was public, this should not be. 

I created #2167 to resolve this. There maybe some carry on affects there so I will do this in a separate PR

Reply:
Thanks for the clarification. I've removed the `public` prefix so the table should be private now.

Reply:
@jumaffre - can you please leave it as public for now. This will conflict with the current work I am doing.